api/v2/openapi.yaml

```diff
@@ -243,6 +243,37 @@ paths:
           $ref: '#/responses/BadRequest'
         '500':
           $ref: '#/responses/InternalServerError'
+  /alertgroups:
```

Comment:
Why not make this a flag or a subpath on the existing API?
This naming can be confusing with the existing `/alerts/groups`.